dashboards/Home_Dashboard.json

```diff
@@ -2210,7 +2210,7 @@
             "tableColumn": "",
             "targets": [
                 {
-                    "expr": "count(node_cpu{mode=\"user\", instance=~\"$host\"}) or count(rdsosmetrics_General_numVCPUs{instance=~\"$host\"})",
+                    "expr": "count(node_cpu{instance=~\"$host\",mode=\"user\"}) or rdsosmetrics_General_numVCPUs{instance=~\"$host\"}",
                     "format": "time_series",
                     "instant": true,
                     "interval": "5m",
```

Comment:
@adivinho Why `5m` and not `$interval`?

Reply:
AFAIR It was done in order to slightly reduce time of the dashboard loading.

Reply:
Please change mode=\"user\" to mode=~\"user\". Otherwise the first part of the formula will return mode as an element in reply. It will cause a conflict with a reply from the second part because there will be no elements.

Reply:
I can confirm that: https://pmmdemo.percona.com/prometheus/graph?g0.range_input=1h&g0.expr=1-absent(node_cpu%7Binstance%3D%22rds-mysql56%22%2Cmode%3D%22user%22%7D)&g0.tab=1&g1.range_input=1h&g1.expr=1-absent(node_cpu%7Binstance%3D%22rds-mysql56%22%2Cmode%3D~%22user%22%7D)&g1.tab=1

But why? What part of the PromQL triggers that behavior?

Reply:
https://prometheus.io/docs/prometheus/latest/querying/functions/#absent()

> In the second example, absent() tries to be smart about deriving labels of the 1-element output vector from the input vector.

🤢

Reply:
Please set mode=~\"user\ in the first formula too.
It's possible that only an RDS host is selected in hosts drop-down menu.

Reply:
Good catch. Wrong copy&paste.